tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -81,11 +81,27 @@ edpm_nodes:
         subnetName: subnet1
       - name: storagemgmt
         subnetName: subnet1
+edpm_nodes_networker:
```

Comment:
I'm not sure we should add this default. IIUC we won't test the networker use case with standalone right? In that case this variable would be wrong, and the code is already checks if `edpm_nodes_networker` is defined, so it should not break other use cases

Reply:
this is correct, with standalone we don't test networked nodes case